ProcessesApi/V1/Boundary/Request/Validation/CreateProcessQueryValidator.cs

```diff
@@ -11,8 +11,6 @@ public CreateProcessQueryValidator()
                             .NotEqual(Guid.Empty);
             // Uncomment when frontend has added relatedEntities as part of the CreateProcess Request
             //RuleFor(x => x.RelatedEntities).NotNull();
```

Comment:
You could create a RelatedEntityValidator that asserts that the ID isn't empty, etc.